nucypher/blockchain/economics.py

```diff
@@ -51,34 +51,36 @@ class BaseEconomics:
     nunits_per_token = 10 ** __token_decimals  # Smallest unit designation
 
     # Period Definition
-    __default_hours_per_period = 24
+    _default_hours_per_period = 24
```

Comment:
Moving to single underscore here for access via subclasses?

Reply:
Maybe I did it wrong, I just thought that these values should be accessible from BaseEconomics._default

Reply:
You want access by *class*  - Is this implemented somewhere access by *instance* cannot be used?  The original intention of double underscore here is to ensure that economic instance attributes are not accidentally mutated during runtime; however, if this is presenting a challenge regarding the inheritance of defaults, it's acceptable to set the to single underscores IMO.

Reply:
I rely on your opinion, so what is the best way to do this?